sn_client/src/audit/tests/setup.rs

```diff
@@ -81,7 +62,11 @@ impl MockNetwork {
 
         if balance > 0 {
             let genesis_pk = GENESIS_CASHNOTE.main_pubkey();
-            self.send(genesis_pk, &owner_pk, balance)
+
+            let genesis_sk_main_pubkey = get_genesis_sk().main_pubkey();
+
+            println!("Sending {balance} from genesis {genesis_pk:?} to {owner_pk:?}");
+            self.send(&genesis_sk_main_pubkey, &owner_pk, balance, false)
                 .map_err(|e| eyre!("failed to get money from genesis: {e}"))?;
```

Comment:
What's the change here? Is `genesis_pk` different from `genesis_sk_main_pubkey`?